README.md

```diff
@@ -91,3 +91,9 @@ See the *parameterization demo* for further explanations of the purpose of these
 
 In order to interpret the attributions produced by an `AttributionMethod`, a few useful visualizers are provided in the `trulens.visualizations` module.
 While the interface of each visualizer varies slightly, in general, the visualizers are a function taking an `np.Array` representing the attributions returned from an `AttributionMethod` and producing an image that can be used to interpret the attributions.
+
+## Citation
+[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.4495857.svg)](https://doi.org/10.5281/zenodo.4495856)
```

Comment:
why are the two ids different? 4495857 vs 4495856

Reply:
The 4495856 one is for all, 4495857, is specifically the 0.0.5 release
From Zenodo:
Cite all versions? You can cite all versions by using the DOI 10.5281/zenodo.4495856. This DOI represents all versions, and will always resolve to the latest one.